tools/train_siamese.py

```diff
@@ -77,3 +77,6 @@
                              model_name=name)
     trainer.train()
     logger.info("Total Training Time:" + str(utils.get_elapsed_time(time.time() - start_time))) 
+
+    if train_config["neighbor_classification"]:
```

Comment:
I think it would be better to create another separate tool script such as `predict_siamese_nn.py`. It doesn't seem to be the best place to do it here because we don't seem to be actually using them, and also because this script should be solely for training. What I usually do is just copy an old script I have (such as `train_siamese.py` in this case), and then gut it out/modify it for the new thing I'm trying to do.